commands/dataset/versions/delete/mod.ts

```diff
@@ -0,0 +1,109 @@
+import { datasetVersions } from "../../../../api/dataset.ts";
+import { datasets } from "../../../../api/dataset.ts";
+import { fields } from "../../../../flags.ts";
+import { asserts } from "../../../../lib/asserts.ts";
+import { dataTable } from "../../../../lib/data-table.ts";
+import { loading } from "../../../../lib/loading.ts";
+import { pickJson } from "../../../../lib/pick-json.ts";
+import { select } from "../../../../prompts/select.ts";
+import { args, command, flags, z } from "../../../../zcli.ts";
+import { defaultFields } from "../mod.ts";
+
+/**
+ * This variable is automatically generated by `zcli add`. Do not remove this
+ * or change its name unless you're no longer using `zcli add`.
+ */
+const subCommands: ReturnType<typeof command>[] = [];
+
+export const del = command("delete", {
```

Comment:
probably a dumb questions but why is this one called `del` and in [projects](https://github.com/Paperspace/cli/pull/63/files#diff-b8a1c4d7a1cd497383078cb13b5504a8be6f9fb763cb7d38643278a8d434ff42R16) is called `_delete`